code/game/objects/items/defib.dm

```diff
@@ -280,16 +277,13 @@
 	worn_icon_state = "defibcompact"
 	w_class = WEIGHT_CLASS_NORMAL
 	slot_flags = ITEM_SLOT_BELT
+	action_slots = ITEM_SLOT_BELT
 	paddle_state = "defibcompact-paddles"
 	powered_state = "defibcompact-powered"
 	charge_state = "defibcompact-charge"
 	nocell_state = "defibcompact-nocell"
 	emagged_state = "defibcompact-emagged"
 
-/obj/item/defibrillator/compact/item_action_slot_check(slot, mob/user)
-	if(slot & user.getBeltSlot())
-		return TRUE
```

Comment:
Pretty sure this is the only use of `getBeltSlot()` so if this is gone it can probably be deleted

Reply:
This is intended to support holoparasites using defibs, I believe, but I'm fairly certain defib code blocks them anyways